biztalk/core/high-availability-using-sql-server-always-on-availability-groups.md

```diff
@@ -269,15 +259,14 @@ If you are adding other BizTalk databases to the BizTalk Databases Backup job, b
 These limitations are for BizTalk Server, SQL Server AlwaysOn Availability Group, and Azure Virtual Machines. These limitations may or may not get addressed in future. 
 
 * Logins, SQL Agent Jobs, the SQL DB Mail profile, and accounts are not managed within Availability Groups. This requires manual modification in Jobs to make sure they run against the primary replica. 
-* SQL Server Analysis Services and SQL Server Integration Services do not participate in Availability Groups. Without this support from SQL Server, there is no HA solution for these in Azure Virtual Machines. BizTalk Server’s BAM capabilities are dependent on these services. 
-* Prior to SQL Server 2016 SP2, Availability Groups don't support MSDTC between databases on the same SQL instance. Therefore, a minimum 8 SQL instances are required to configure BizTalk. 
+* SQL Server Analysis Services and SQL Server Integration Services do not participate in Availability Groups. Without this support from SQL Server, there is no HA solution for these in Azure Virtual Machines. BizTalk Server’s BAM capabilities are dependent on these services.
 
-  Starting with SQL Server 2016 SP2 *and* BizTalk Server 2016 [CU5](https://support.microsoft.com/help/2555976/service-pack-and-cumulative-update-list-for-biztalk-server), the BizTalk databases can use the same SQL Server instance. 
+  BizTalk Server 2020 supports high availability for BAM DTS packages using SSIS Catalog.
```

Comment:
Please see the markdown guidance for adding a note - I think each line needs to start with a '>' :

## Add a note
> [!NOTE]
> Type in your text. Instead of "NOTE", you can also use WARNING, IMPORTANT, and TIP.